docker-compose.dev.yml

```diff
@@ -27,7 +27,7 @@ services:
       context: ./server
       dockerfile: Dockerfile.dev
       args:
-        VERSION: ${LAST_COMMIT}
+        VERSION: '1'
```

Comment:
Is this supposed to be in this PR?

Reply:
As of this morning, whenever I try to build this locally, it will fail unless I have a hardcoded string for the Version. I'm not sure whats going on here. I'll remove it for now.